### PR TITLE
New Object EVTB based on AFF 

### PR DESCRIPTION
**Pull request by rauva**
See #5708 

https://github.com/abapGit-tests/EVTB

### CODE REVIEW

File: src/objects/aff/zcl_abapgit_object_evtb.clas.abap [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    CONSTANTS:
+    co_table_name TYPE tabname VALUE 'EVTB_HEADER'.
+ENDCLASS.
+
+
+
+CLASS zcl_abapgit_object_evtb IMPLEMENTATION.
+
+  METHOD zif_abapgit_object~serialize.
+    IF ms_item-obj_type <> 'EVTB'.
```
Comment (mbtools) [inline src/objects/aff/zcl_abapgit_object_evtb.clas.abap:26]: the ifs checking the `obj_type` are not necessary. `zcl_abapgit_object_evtb` will only be called for objects of type `EVTB` so the ifs are always passed.

Overall, no need to redefine the ser/deser methods.

Comment (mbtools) [reply src/objects/aff/zcl_abapgit_object_evtb.clas.abap:26]: PS: Same is true for some of the other AFF object types that contain similar ifs.

**Review by mbtools — DISMISSED** [review]
simplify

**Comment by larshp** [comment]
see changes in https://github.com/abapGit/abapGit/pull/5712

File: src/objects/aff/zcl_abapgit_object_evtb.clas.abap [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+        lv_where_expr_active      = 'EVTB_NAME = '   &&  cl_abap_dyn_prg=>quote( ms_item-obj_name )  &&
+                                 ' AND VERSION = '   &&  cl_abap_dyn_prg=>quote( 'A' ).
+        lv_where_expr_inactive    = 'EVTB_NAME = '   &&  cl_abap_dyn_prg=>quote( ms_item-obj_name ) &&
+                                 ' AND VERSION = '   &&  cl_abap_dyn_prg=>quote( 'I' ).
+
+        lv_select                 = 'CHANGED_BY'.
+
+
+        SELECT SINGLE (lv_select) INTO @lv_user
```
Comment (mbtools) [inline src/objects/aff/zcl_abapgit_object_evtb.clas.abap:72]: `@lv_user` -> `lv_user`

File: src/objects/aff/zcl_abapgit_object_evtb.clas.abap [hunk 3]
```diff
@@ -0,0 +1,62 @@
+CLASS zcl_abapgit_object_evtb DEFINITION
+  PUBLIC
+  INHERITING FROM zcl_abapgit_object_common_aff
+
```
Comment (larshp) [inline src/objects/aff/zcl_abapgit_object_evtb.clas.abap:4]:
```suggestion
```

File: src/objects/aff/zcl_abapgit_object_evtb.clas.abap [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+CLASS zcl_abapgit_object_evtb IMPLEMENTATION.
+
+  METHOD zif_abapgit_object~changed_by.
+
+    DATA: lv_user                TYPE string,
+          lx_error               TYPE REF TO cx_root,
+          lv_where_expr_active   TYPE string,
+          lv_where_expr_inactive TYPE string,
+          lv_select              TYPE string.
```
Comment (larshp) [inline src/objects/aff/zcl_abapgit_object_evtb.clas.abap:25]:
```suggestion
          lx_error               TYPE REF TO cx_root.
```

**Review by larshp — APPROVED** [review]
looks good to me

**Comment by larshp** [comment]
welcome to abapGit as a contributor 🎉

**Pull request merged**